lib/AppInfo/Application.php

```diff
@@ -91,9 +91,8 @@
 
 final class Application extends App implements IBootstrap {
 	public const APP_ID = 'mail';
-	public const CONTEXT_CHAT_MESSAGE_MAX_AGE = 60 * 60 * 24 * 365.25; // 1 year
```

Comment:
The .25 is to account for leap years.

Also, the comments should be kept for clarity.